templates/default/node/nodelist.html

```diff
@@ -5,153 +5,159 @@
 <H1>{$layout.pagetitle}</H1>
 {$lastonline_limit = ConfigHelper::getConfig('phpui.lastonline_limit')}
 <FORM method="get" action="?m={$layout.module}" name="choosefilter">
-<p style="display: none;">
```

Comment:
Czy ten rewrite zmienia coś poza tabulatorami? Jeśli nie to proponuję nie ruszać formatowania, bo potem ciężko portować inne zmiany.